app/routes.py

```diff
@@ -23,48 +25,60 @@
 app.route = prefix_route(app.route, ROUTE_PREFIX)
 
 
+def make_api_compliant_response(response_object: dict):
```

Comment:
Although I like the python typing syntax, we don't use it everywhere else, therefore remove `: dict`

Reply:
so we don't use `->` syntax neither?
too bad, it adds a little bit of documentation (and type forcing) in a neatly way

Reply:
my last (forced)push has invalidated your last review, could you give it another look?